<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbEnumerations</class>
 <widget class="QMainWindow" name="dbEnumerations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BiblioteQ: Database Enumerations Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/book.png</normaloff>:/book.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QFrame" name="frame">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget_11">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Book Binding Types</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="bookBindingsList">
              <property name="tabKeyNavigation">
               <bool>true</bool>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <spacer name="horizontalSpacer_13">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="addBookBinding">
                <property name="toolTip">
                 <string>Add</string>
                </property>
                <property name="icon">
                 <iconset resource="../Icons/icons.qrc">
                  <normaloff>:/16x16/add.png</normaloff>:/16x16/add.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="removeBookBinding">
                <property name="toolTip">
                 <string>Remove</string>
                </property>
                <property name="icon">
                 <iconset resource="../Icons/icons.qrc">
                  <normaloff>:/16x16/eraser.png</normaloff>:/16x16/eraser.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>16</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_14">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="_2">
      <item>
       <widget class="QToolButton" name="saveButton">
        <property name="toolTip">
         <string>Save Changes</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/32x32/save.png</normaloff>:/32x32/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+S</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="reloadButton">
        <property name="toolTip">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/32x32/reload.png</normaloff>:/32x32/reload.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="cancelButton">
        <property name="toolTip">
         <string>Close Browser</string>
        </property>
        <property name="icon">
         <iconset resource="../Icons/icons.qrc">
          <normaloff>:/32x32/cancel.png</normaloff>:/32x32/cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+W</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bookBindingsList</tabstop>
  <tabstop>addBookBinding</tabstop>
  <tabstop>removeBookBinding</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>reloadButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
